controller/engine_controller.go

```diff
@@ -529,7 +530,7 @@ func (ec *EngineController) deleteInstanceWithCLIAPIVersionOne(e *longhorn.Engin
 			pod = nil
 		}
 
-		ec.logger.WithField("engine", e.Name).Debug("Prepared to delete engine pod because of outdated version")
+		ec.logger.WithField("engine", e.Name).Debug("Preparing to delete engine pod because of outdated version")
```

Comment:
`Deleting engine pod because of outdat....`

Reply:
Modified.